src/systems/ec/implementations/react/packages/social-media-share/package.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Please make the package private ;-) (check the other React packages, they're all private now)